nsis/SectionCoreReqs.nsh

```diff
@@ -18,8 +18,8 @@
   
   SetOutPath "$INSTDIR"
   !insertmacro UNINSTALL.LOG_OPEN_INSTALL
-    File           /oname=${APP_EXE}                ..\bin\pcsx2.exe
-    ;File /nonfatal /oname=pcsx2-dev.exe  					..\bin\pcsx2-dev.exe
+    File			..\bin\pcsx2.exe
```

Comment:
Probably better to use spaces instead of tabs. Same goes for several of the other changes as well.